shared/src/main/scala/minitest/runner/ConsoleLogger.scala

```diff
@@ -25,14 +25,14 @@ final class ConsoleLogger extends Logger {
 
   def ansiCodesSupported(): Boolean =
     withColors
-  def error(msg: String): Unit =
+  def error(msg: String | Null): Unit =
```

Comment:
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.
Occurences in class: 5